docs/Troubleshooting.md

```diff
@@ -170,6 +171,20 @@ Now perform the operation in the app that triggers the problem. Hopefully this
 will reveal some error messages in the **Console** area of the panel. If this
 is the case, please take a screenshot and include it with your issue.
 
+## Logs
+
+Logs are generated by the Brim app and the `zqd` process it launches to act as
```

Comment:
Maybe...

> Logs are generated by the Brim app and the `zqd` process it launches (the engine for querying your data).

Reply:
@jameskerr: Yeah, I agree that's an improvement. I've pushed a commit with that change. Thanks for the approval!